ci-operator/config/redhat-cps/front-proxy/redhat-cps-front-proxy-main.yaml

```diff
@@ -43,3 +49,19 @@ zz_generated_metadata:
   branch: main
   org: redhat-cps
   repo: front-proxy
+images:
+- dockerfile_literal: |-
+    FROM quay.io/centos/centos:stream8
+    ADD cps-front-proxy /usr/bin/cps-front-proxy
+    ENTRYPOINT ["/usr/bin/cps-front-proxy"]
+  from: os
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/redhat-cps/front-proxy/bin/cps-front-proxy
+  to: cps-front-proxy
+promotion:
+  namespace: kcp
```

Comment:
@stevekuznetsov do you think this is reasonable or should we create a new org for redhat-cps?

I'm thinking if it's just this one image, having `registry.ci.openshift.org/kcp/cps-front-proxy` may be reasonable?